src/main/resources/templates/govuk/button.ftl

```diff
@@ -9,3 +9,8 @@
     </button>
   </#if>
 </#macro>
+
+<#macro buttonWithSecondaryAction primaryButtonText secondaryButtonText primaryButtonClass="" primaryButtonUrl="#" secondaryButtonClass="govuk-button--tertiary" secondaryButtonUrl="#" buttonsDisabled=false>
```

Comment:
Wrap buttons in `<div class="govuk-button-group">` per https://design-system.service.gov.uk/components/button/secondary-combo/index.html (it spaces them out a bit more)